### PR TITLE
Fix #19: update readme.md

### LINKED ISSUE #19: The readme sucks

### PR DESCRIPTION
**Pull request by rileyjshaw**
This is a first pass at an updated README, re: issue #19.

I didn’t go into depth for the **Contributing** section, since I’m still trying to figure that out. But once you update that, I will read it, and then start contributing in that way.

I usually like to call out the license from the README, but we can delete that part if it’s not relevant.

And speaking of deletion, please feel free to cut this up and rewrite it however you see fit! I hope it helps to have an initial draft in place, but that’s all this needs to be.

…can’t wait to commit--blog this 😂

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+
+If you use Git, you’ve already begun! To create a public **commit --blog** of your own, you don’t need to dive deeper into this repo. Go to https://commit--blog.com, follow the instructions there to create an account, and start picking some of your favourite commits to publish.
+
+### Example blogs
+
+To see what a **commit --blog** can look like, check out some active blogs:
+
+- https://uniphil.commit--blog.com
+- https://rileyjshaw.commit--blog.com
+- Add your own by [submitting a pull request](https://github.com/uniphil/commit--blog/edit/master/readme.md)!
```
Comment (uniphil) [inline readme.md:17]: THANK YOU 🥰🥰🥰🥰🥰

this is perfect, and lacking this part of things is exactly what I meant about the readme sucking before, and this is exactly where i get stuck on trying to put words together. This is perfect ❤️

**Comment by uniphil** [comment]
This is wonderful!!!!!!

I'm _glad_ you didn't go into depth on contributing, because I want to fix a few things up there which will change the process a lot anyway. That's near the top of my list :)

License-- yes. I think it's been GPL3 in my head? Might need to spend a minute thinking about it, but yes, def want to fix that.

**Comment by uniphil** [comment]
> But once you update that, I will read it, and

😯 hey.... you wanna... see if the setup steps in the readme make any sense now? ❤️

**Comment by rileyjshaw** [comment]
> > But once you update that, I will read it, and
> 
> 😯 hey.... you wanna... see if the setup steps in the readme make any sense now? ❤️

image

OOOOOMMMMMMGGGGGGGGG!!!!!!

My secret fear is setting up new Python projects, because it invariably leads to half a day clambering around virtual virtual virtualenv envspace and breaking my laptop.

But setting this up was A BREEZE!!!!!!!

Tiny PR inbound but this is epic.

**Comment by rileyjshaw** [comment]
@uniphil oh also, I lolled at:

> Successfully installed [approximately 1 million packages]

**Comment by uniphil** [comment]
AW YEAAAAAA YAYYYYYYYYYYYYY!!!! I'm so happy it worked (mod one typo oops/thanks!)

> and breaking my laptop

^^ this is toooooooooooo real hahaaa..... 😭

**Pull request merged**